### ISSUE TITLE
Could you provide with commonly used pretrained models with se modules?

### ISSUE THREAD

**Issue by KleinXin** [Alibaba-MIIL/ImageNet21K#15]
As it is evident SE module can dramatically improve the accuracy of network such as ResNet, coule you please also provide some pretrained models?

Just like SE-ResNet101/SE-ResNet50... in this page.

https://github.com/Cadene/pretrained-models.pytorch#torchvision

**Comment by mrT23** [comment]
TResNet models have SE, and are better the SE-ResNet. use them.
(TResNet-M is equivalent in runtime to SE-ResNet50, TResNet-L-V2  is equivalent in runtime to SE-ResNet101)

note that mobilenet and OFA models also use SE.
